lib/components/viewers/route-viewer.js

```diff
@@ -62,7 +67,8 @@ class RouteViewer extends Component {
       : []
     const agencySortedRoutes = transitOperators.length > 0
       ? sortedRoutes.sort((a, b) => {
-        return operatorIndexForRoute(transitOperators, a) - operatorIndexForRoute(transitOperators, b)
+        return routeOperatorComparatorValue(transitOperators, a) -
+          routeOperatorComparatorValue(transitOperators, b)
```

Comment:
I wonder if we should just be embedding this logic within otp-ui? Perhaps not, but I think the question is worth asking.

Reply:
This seems specific to the route-viewer, so I'd say if the route-viewer becomes an otp-ui component, then it'd make sense to put this in otp-ui core-utils.